README.md

```diff
@@ -424,6 +424,19 @@ The following boards are pre-integrated.
 
 > To help you know if you have to worry, we'll call such boards "pre-integrated" and prefix each section with suitable guidance.
 
+## PlatformIO
+
+For use with PlatformIO, the `lmic_project_config.h` has to be disabled with the flag `ARDUINO_LMIC_PROJECT_CONFIG_H_SUPPRESS`.  
+The settings are defined in PlatformIO by `build_flags`.  
+```ini
+lib_deps =
+    MCCI LoRaWAN LMIC library
+
+build_flags =
+    -D ARDUINO_LMIC_PROJECT_CONFIG_H_SUPPRESS
+    -D CFG_eu868=1
+    -D CFG_sx1276_radio=1
+```
 ## Manual configuration
```

Comment:
Thanks for this contribution.

markdownlint wants a blank line before line 431, and after line 439. Add that, and we can commit.

(Then I'll start asking if we can add the json file to configure the library for install from PlatformIO's UI.)

Reply:
Hi i have added the blank lines.

Reply:
Thanks!